source/detail/parser/x86_64.cpp

```diff
@@ -339,6 +334,17 @@ namespace smeagle::x86_64 {
     return regString;
   }
 
+  // Get a single string of locations from register classes
+  std::string getRegistersString(std::pair<RegisterClass, RegisterClass> regClasses, int order) {
```

Comment:
I think I mentioned this to Matt in a meeting (how to represent multiple registers for one string) and he said to make something up, so it's probably okay for now.

Reply:
Sounds good. I think we'll need a couple of extra cases when we get to aggregates since the registers will be possibly more spread out than they are here, but we can cross that bridge at that time.